pom.xml

```diff
@@ -49,6 +53,7 @@
     <maven.compiler.parameters>true</maven.compiler.parameters>
     <quarkus.platform.version>2.3.0.Final</quarkus.platform.version>
     <version.surefire.plugin>3.0.0-M5</version.surefire.plugin>
+    <jrfDocUrl>https://bestsolution-at.github.io/jfr-doc/openjdk-17.json</jrfDocUrl>
```

Comment:
Is this used anywhere actually? Or is it here for documentation purposes? Good to have it in any case.

Reply:
Yes, I'm using it in JrfUnitGeneratorProcessor in order to retrieve the endpoint `System.getProperties().get("jfrDocUrl")` the objective is to provide a default value that can be overridden if needed during the release pipeline, `mvn clean install -DjfrDocUrl=http...`